raft/log.go

```diff
@@ -1095,18 +1220,18 @@ func (l *Log) internalApply(typ LogEntryType, command []byte) (index uint64, err
 }
 
 // Wait blocks until a given index is applied.
-func (l *Log) Wait(index uint64) error {
+func (l *Log) Wait(idx uint64) error {
 	// TODO(benbjohnson): Check for leadership change (?).
```

Comment:
I'd like to rework the `Wait()` so it doesn't poll so I'm not sure that it's worth making tickets.